validate/application.py

```diff
@@ -0,0 +1,26 @@
+from flask import Flask, request
+from hashlib import sha512
+from tests import check_payload, get_payload, valid_signature
```

Comment:
Let's just keep everything in one script for simplicity, especially that we're getting rid of most of these?